### PR TITLE
[VBLOCKS-2479] fix(js): use common constants for keying quality warnings

### PR DESCRIPTION
**Pull request by mhuynh5757**
## Submission Checklist

 - [x] Updated the `CHANGELOG.md` to reflect any **feature**, **bug fixes**, or **known issues** made in the source code
 - [x] Tested code changes and observed expected behavior in the example app
 - [x] Performed a visual inspection of the `Files changed` tab prior to submitting the pull request for review to ensure proper usage of the style guide

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

## Description

This PR fixes missing call quality events by properly using the common constants to key the native info objects that are passed to the JS layer.

## Breakdown

- Use common constants to key the native info objects.

## Validation

- Manually tested with the test app.

## Additional Notes

N/A

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ Twilio Voice React Native SDK has now reached milestone `beta.5`. Included in th
 
 - Fixed and improved the docstrings for the `Voice` and `Call` listeners. The descriptions of the events and listeners should now point to the correct docstrings.
 
+## Fixes
+
+- Call quality warning events should now properly pass arguments to listener functions.
+
```
Comment (charliesantos) [inline CHANGELOG.md:13]: The item above this is also a "fix". Should we combine the two?

**Review by charliesantos — APPROVED** [review]
Left a minor comment. You can merge once it's addressed.

**Review by charliesantos — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,15 +19,13 @@ Twilio Voice React Native SDK has now reached milestone `beta.5`. Included in th
...
   ```
 
-## Fixes
-
-### Platform Specific Features
+### Platform Specific Changes
 
 #### Android
-- Call timestamp now in RFC-822 format, not stored as a double from epoch
+- Call timestamp now in simplified ISO-8601 format, not stored as a double from epoch
```
Comment (kpchoy) [inline CHANGELOG.md:25]:
```suggestion
- Call timestamp now in simplified ISO-8601 format, not stored as a double from epoch.
```

File: test/app/e2e/call.test.ts [hunk 3]
```diff
@@ -47,19 +48,49 @@ describe('call', () => {
...
-      await waitFor(element(by.text('Call State: disconnected')))
-        .toBeVisible()
-        .withTimeout(DEFAULT_TIMEOUT);
+      it('should log a constant-audio-input-level quality warning', async () => {
+        await element(by.text('CONNECT')).tap();
+        await waitFor(element(by.text('Call State: connected')))
+          .toBeVisible()
+          .withTimeout(DEFAULT_TIMEOUT);
+
+        // Let the call go for 15 seconds
```
Comment (kpchoy) [inline test/app/e2e/call.test.ts:73]: why does this call go for 15sec, but the other goes for 5sec?

**Review by kpchoy — CHANGES_REQUESTED** [review]

File: test/app/e2e/call.test.ts (hunk 3 above)
Comment (mhuynh5757) [reply test/app/e2e/call.test.ts:73]: We need this one to go long enough to issue the call quality warnings. The other one only connects and disconnects.

**Review by kpchoy — APPROVED** [review]

**Pull request merged**